pageserver/src/layered_repository.rs

```diff
@@ -393,9 +393,23 @@ impl Repository for LayeredRepository {
 
     fn detach_timeline(&self, timeline_id: ZTimelineId) -> anyhow::Result<()> {
         let mut timelines = self.timelines.lock().unwrap();
+        // check no child timelines, because detach will remove files, which will brake child branches
+        // FIXME this can still be violated because we do not guarantee
+        //   that all ancestors are downloaded/attached to the same pageserver
+        dbg!(&timeline_id);
```

Comment:
`dbg!` (here and below) looks like something worth removing

Reply:
Cleaned it up, thanks!

Reply:
That comment did disappear after I hit `Refresh` 🤔